### DISCUSSION TITLE
Visualize HTML

### DISCUSSION THREAD

**Discussion by fdemiguel** [singerdmx/flutter-quill#1464] [Web]
Hi everyone!

I have information coming from a database in HTML format, what is the best solution currently to display it in the Quill editor?

Thanks.

**Comment by EchoEllet** [comment]
Take a look at [flutter_quill_delta_from_html](https://pub.dev/packages/flutter_quill_delta_from_html). Keep in mind converting **HTML** to **Delta** doesn't work always as expected and should be avoided if you want to store new data in the database.

**Reply by Philipp1470** [reply]
Response #2

**Comment by Philipp1470** [comment]
Response #1

**Comment by Philipp1470** [comment]
Response #2

**Comment by EchoEllet** [comment]
See [Conversion to HTML](https://github.com/singerdmx/flutter-quill#-conversion-to-html) section for details.
